.github/workflows/deploy.yaml

```diff
@@ -195,3 +195,119 @@ jobs:
           export ROUTE="$(oc get route ${API} -o jsonpath='{.spec.host}')"
           echo "${API} is exposed at 'https://'${ROUTE}" 
 
+   # Deploy App and Api images in Test
+  deployTest:
+    name: Deploy APP to Test environment
```

Comment:
Another change I suggest we do is to tie these jobs to a Github environment so we can easily keep track of what's been deployed where, and we can also have separate secrets for each environment so we can move all of the variables in there
```
name: Deploy APP to Test environment
environment: 
  name: Development
  url: 'https://digital-journeys-dev.apps.silver.devops.gov.bc.ca'
```

Reply:
Example of what this gives us: https://dev.to/n3wt0n/everything-you-need-to-know-about-github-actions-environments-9p7